tests/a_scripts/__init__.py

```diff
@@ -0,0 +1,2 @@
+# Copyright 2022 MosaicML LLM Foundry authors
```

Comment:
why is it `a_scripts`?

Reply:
Bit of a hack right now - mysterious gpu failure if script tests run last https://github.com/mosaicml/llm-foundry/actions/runs/7055479688/job/19206776145

Reply:
Everything looks green.. going to add a comment and come back to this later 🙂